usr/src/cmd/zpool/zpool_vdev.c

```diff
@@ -561,6 +561,8 @@ get_replication(nvlist_t *nvroot, boolean_t fatal)
 	    &top, &toplevels) == 0);
 
 	lastrep.zprl_type = NULL;
```

Comment:
Are these 3 lines just to silence the compiler? It's already defined to be {0} previously.